taxonomy/utils.py

```diff
@@ -157,6 +205,68 @@ def refresh_course_skills(courses, should_commit_to_db):
     )
 
 
+def refresh_program_skills(programs, should_commit_to_db):
+    """
+    Refresh the skills associated with the provided programs.
+    """
+    all_failures = []
+    success_programs_count = 0
+    skipped_programs_count = 0
+
+    for index, program in enumerate(programs, start=1):
+        program_overview = program['overview']
+        if program_overview:
+            try:
+                program_skills = get_emsi_skills_data(program_overview, index)
```

Comment:
I think the program skills would be more accurate if they are a union of the skills associated with the courses within the program ... no?

Reply:
We used this approach because of existing edX programs and 2U degrees (which are basically derived from Program). In case of 2U, we will be creating an empty program (i.e. without courses) so taking a union out of it's courses may not be possible in every case.